src/lib/Index.ts

```diff
@@ -43,13 +43,21 @@ class Index {
     }
 
     // http://w3c.github.io/IndexedDB/#retrieve-multiple-referenced-values-from-an-index
-    public getAllKeys(range: FDBKeyRange, count?: number) {
+    public getAllKeys(
+        range: FDBKeyRange,
+        count?: number,
+        direction?: FDBCursorDirection,
+    ) {
         if (count === undefined || count === 0) {
             count = Infinity;
         }
 
         const records = [];
-        for (const record of this.records.values(range)) {
+        const values =
+            direction === undefined
+                ? this.records.values(range)
+                : this.records.values(range, direction);
+        for (const record of values) {
```

Comment:
Make this `for (const record of this.records.values(range, direction)) {` for consistency with the others.